dockerfiles/che/entrypoint.sh

```diff
@@ -280,6 +280,18 @@ init() {
   export JAVA_OPTS="${JAVA_OPTS} -Dche.docker.network=$NETWORK_NAME"
 }
 
+add_cert_to_truststore() {
+
+    if [ "${OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE}" != "" ]; then
+        if [ ! -f /data/openshift.crt ]; then
+            echo "Found a custom cert. Adding it to java trust store..."
+            echo "${OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE}" > /data/openshift.crt
+            echo yes | keytool -keystore /data/openshift.jks -importcert -alias HOSTDOMAIN -file /data/openshift.crt -storepass minishift
+        fi
+            export JAVA_OPTS="${JAVA_OPTS} -Djavax.net.ssl.trustStore=/data/openshift.jks -Djavax.net.ssl.trustStorePassword=minishift"
```

Comment:
Because of formatting looks like this line is related to previous conditional section. Please reformat,